extensions/jsonb/deployment/src/main/java/io/quarkus/jsonb/deployment/JsonbProcessor.java

```diff
@@ -71,6 +73,9 @@ void build(BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
         serviceProvider.produce(new ServiceProviderBuildItem(JsonbComponentInstanceCreator.class.getName(),
                 QuarkusJsonbComponentInstanceCreator.class.getName()));
 
+        // Accessed in jakarta.json.bind.spi.JsonbProvider.provider()
+        serviceProvider.produce(new ServiceProviderBuildItem(JsonbProvider.class.getName()));
```

Comment:
Could we use the same approach we used here: https://github.com/quarkusio/quarkus/pull/44649/files .

(and I suspect we can then drop the registration of `JsonBindingProvider` a bit above)

Reply:
Done